src/app/containers/StoryPromo/index.jsx

```diff
@@ -132,15 +132,20 @@ const StoryPromoContainer = ({
 
   const useLargeImages = promoType === 'top' || promoType === 'leading';
 
+  const headingTagOverride = promoType === 'recommendation' ? 'div' : null;
+
+  const headlineProps = {
+    script,
+    service,
+    promoType,
+    promoHasImage: displayImage,
+    ...(headingTagOverride && { as: headingTagOverride }),
+  };
+
   const Info = (
     <>
       {headline && (
-        <Headline
-          script={script}
-          service={service}
-          promoType={promoType}
-          promoHasImage={displayImage}
-        >
+        <Headline {...headlineProps}>
```

Comment:
This looks good. Just one comment. I think leaving the `Headline` props as they were and adding `as` would be cleaner.
```jsx
<Headline 
  script={script}	
  service={service}	
  promoType={promoType}	
  promoHasImage={displayImage}
  as={headingTagOverride} >
```
If `hedingTagOverride` is null, the `as` wont be applied.